atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientFactory.java

```diff
@@ -96,19 +97,17 @@ private CassandraClient instrumentClient(Client rawClient) {
         return client;
     }
 
-    private Cassandra.Client getRawClientWithKeyspace(
-            InetSocketAddress inetSocketAddress, CassandraKeyValueServiceConfig kvsConfig) throws TException {
-        Client ret = getRawClient(inetSocketAddress, kvsConfig, sslSocketFactory);
+    private Cassandra.Client getRawClientWithKeyspace() throws TException {
```

Comment:
after your refactor, the name of this method doesn't really make sense. maybe `getRawClientWithKeyspaceSet`?

Reply:
that's reasonable, sure